.gitignore

```diff
@@ -4,7 +4,6 @@ app/release
 *patch
 *.iml
 .gradle
-/local.properties
```

Comment:
local.properties shouldn't be commited to git. Its mean't to be local only

Reply:
Fixed with latest commit.